GPU/GLES/TextureCacheGLES.cpp

```diff
@@ -341,13 +341,15 @@ class TextureShaderApplier {
 };
 
 void TextureCacheGLES::ApplyTextureFramebuffer(VirtualFramebuffer *framebuffer, GETextureFormat texFormat, FramebufferNotificationChannel channel) {
-	DepalShader *depal = nullptr;
+	DepalShader *depalShader = nullptr;
 	uint32_t clutMode = gstate.clutformat & 0xFFFFFF;
 	bool need_depalettize = IsClutFormat(texFormat);
 
-	bool useShaderDepal = framebufferManager_->GetCurrentRenderVFB() != framebuffer && (gstate_c.Supports(GPU_SUPPORTS_GLSL_ES_300) || gstate_c.Supports(GPU_SUPPORTS_GLSL_330));
+	bool depth = channel == NOTIFY_FB_DEPTH;
+	bool useShaderDepal = framebufferManager_->GetCurrentRenderVFB() != framebuffer && (gstate_c.Supports(GPU_SUPPORTS_GLSL_ES_300) || gstate_c.Supports(GPU_SUPPORTS_GLSL_330)) && !depth;
 	if (!gstate_c.Supports(GPU_SUPPORTS_32BIT_INT_FSHADER)) {
 		useShaderDepal = false;
+		depth = false;  // Can't support this
```

Comment:
Hm, is it better to use the color part of the framebuf or just ignore the attachment I wonder?

-[Unknown]

Reply:
The color part probably isn't usually very meaningful when you're trying to read the depth, so ignoring is probably better...  unfortunately currently no good way to bind an empty texture here? Hm.

Reply:
I guess the better thing would be a Supports flag and handle this when choosing candidates...

-[Unknown]

Reply:
Yeah, I think we already do...